packages/core/forms/src/composables/useAbstractFields.ts

```diff
@@ -86,19 +87,25 @@ export default function useAbstractFields(formData: AbstractFieldParams) {
   const validate = (calledParent?: any) => {
     clearValidationErrors()
 
-    const schemaValidator = formData.schema?.validator
     const validateAsync = objGet(formData.formOptions, 'validateAsync', false)
     let results: any[] = []
 
-    if (schemaValidator && formData.schema.readonly !== true && formData.disabled !== true) {
+    if (formData.schema.readonly !== true && formData.disabled !== true) {
       const validators = []
 
-      if (!Array.isArray(schemaValidator)) {
-        validators.push(convertValidator(schemaValidator))
-      } else {
-        forEach(schemaValidator, validator => {
-          validators.push(convertValidator(validator))
-        })
+      const schemaValidator = formData.schema?.validator
+      if (schemaValidator) {
+        if (!Array.isArray(schemaValidator)) {
+          validators.push(convertValidator(schemaValidator))
+        } else {
+          forEach(schemaValidator, validator => {
+            validators.push(convertValidator(validator))
+          })
+        }
+      }
+
+      if (formData.externalValidator && typeof formData.externalValidator === 'function') {
```

Comment:
I saw the first line we imported `isFunction` here can we utilize that here instead of using `typeof`?